src/main/java/htmlpublisher/HtmlPublisherTarget.java

```diff
@@ -79,29 +80,36 @@
     private static final String WRAPPER_NAME = "htmlpublisher-wrapper.html";
 
     /**
-     * @deprecated Use {@link #HtmlPublisherTarget(java.lang.String, java.lang.String, java.lang.String, boolean, boolean, boolean)}. 
+     * @deprecated Use {@link #HtmlPublisherTarget(java.lang.String, java.lang.String, java.lang.String, java.lang.String, boolean, boolean, boolean)}.
      */
     @Deprecated
     public HtmlPublisherTarget(String reportName, String reportDir, String reportFiles, boolean keepAll, boolean allowMissing) {
-        this(reportName, reportDir, reportFiles, keepAll, false, allowMissing);
+        this(reportName, reportDir, reportFiles, "",keepAll, false, allowMissing);
+    }
+
+    public String getReportTitles() {
+        return reportTitles;
     }
 
     /**
      * Constructor.
      * @param reportName Report name
      * @param reportDir Source directory in the job workspace
      * @param reportFiles Files to be published
+     * @param reportTitles Files Title to be published
      * @param keepAll True if the report should be stored for all builds
      * @param alwaysLinkToLastBuild If true, the job action will refer the latest build.
      *      Otherwise, the latest successful one will be referenced
      * @param allowMissing If true, blocks the build failure if the report is missing
      * @since 1.4
+
      */
     @DataBoundConstructor
-    public HtmlPublisherTarget(String reportName, String reportDir, String reportFiles, boolean keepAll, boolean alwaysLinkToLastBuild, boolean allowMissing) {
+    public HtmlPublisherTarget(String reportName, String reportDir, String reportFiles,String reportTitles, boolean keepAll, boolean alwaysLinkToLastBuild, boolean allowMissing) {
```

Comment:
I believe if we change this signature, it will cause problems for any plugin using this signature (perhaps Job DSL / Pipeline). I think the correct thing to do is create a new constructor. @jglick has helped me out before, can you remind me what the best practice is for this?